scripts/repair_batlas_references.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+import argparse
+import itertools
+import json
+import sys
+import transaction
+
+from pleiades.dump import getSite, spoofRequest
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Repair old-style and idiosyncratic BAtlas references on Place objects"
+    )
+    parser.add_argument("input_file", type=str, help="The JSON input file name")
+    parser.add_argument("--dry-run", action="store_true", help="Enable dry run mode")
+    # Deal with run script wrapping. We're passed a -c flag we don't use:
+    parser.add_argument("-c", type=str, help=argparse.SUPPRESS)
+
+    parsed = parser.parse_args()
+
+    return parsed.input_file, parsed.dry_run
+
+
+def ingest_json_file(file_path):
+    try:
+        with open(file_path, "r") as file:
+            data = json.load(file)
+            return data
```

Comment:
I'd probably move this out of the `with` and maybe get rid of the try/except since we're just re-raising the error in all cases, but this is fine.